### ISSUE TITLE
请问有人跟我一样load不进dev test 的数据吗？

### ISSUE THREAD

**Issue by Yuqi92** [macanv/BERT-BiLSTM-CRF-NER#18]
我试着运行了主程序，可是发现每次都出现dev test数据load不进去的情况，train就没有这样的问题，查了代码觉得没什么问题，请问有人跟我遇到一样的问题吗？能否帮忙解答一下，我可能错过了什么，万分感谢~
![screen shot 2018-12-11 at 11 09 00 pm](https://user-images.githubusercontent.com/14217693/49848320-3ae9ec80-fd9a-11e8-9e71-51b3e4c9f48f.png)
![screen shot 2018-12-11 at 11 09 09 pm](https://user-images.githubusercontent.com/14217693/49848328-40dfcd80-fd9a-11e8-86c1-7a30b8a1132f.png)

**Comment by Little-girl-1992** [comment]
你好， 我也遇到了这个问题  不过我的结果是nan

INFO:tensorflow:*** Features ***
INFO:tensorflow:  name = input_ids, shape = (?, 128)
INFO:tensorflow:  name = input_mask, shape = (?, 128)
INFO:tensorflow:  name = label_ids, shape = (?, 128)
INFO:tensorflow:  name = segment_ids, shape = (?, 128)

INFO:tensorflow:***** Predict results *****
INFO:tensorflow:  eval_f = nan
INFO:tensorflow:  eval_precision = nan
INFO:tensorflow:  eval_recall = nan
INFO:tensorflow:  global_step = 327
INFO:tensorflow:  loss = 2.4771109
